### DISCUSSION TITLE
issues when using processing_lvl >= 4

### DISCUSSION THREAD

**Discussion by wellyowo** [openhsi/openhsi#42] [Q&A] [answered]
HI, we are using OpenHSI library for our team's HSI camera, but we met some problem about change the processing_lvl from 2 to 4. Attch file is the error message we met.
Thanks!
![error_part1](https://user-images.githubusercontent.com/95109566/185553487-18ca2964-c994-4f34-a55e-5ebeb4335468.png)
![error_part2](https://user-images.githubusercontent.com/95109566/185553500-bf028753-ae23-4011-ae33-bb303a6abdc7.png)

**Comment by YiweiMao** [comment, marked as answer]
Hi, just saw this. I was checking the Github issues and RobotX forum instead of these Discussions. 
So sorry for the late reply. 

This is caused by the latest version of `xarray` which made some changes to the way coordinates are indexed and affects files created before that version. You can fix this by using version xarray==2022.3.0 which is the latest version that passes all my tests. 
`pip install xarray==2022.3.0`
